LocalFeeder/dss_functions.py

```diff
@@ -18,21 +19,23 @@ def get_loads(dss, circuit):
         }
         _ = circuit.SetActiveElement("Load.%s" % datum["name"])
         cktElement = dss.CktElement
-        bus = cktElement.BusNames()[0].split(".")
+        buses = cktElement.BusNames()
+        bus = buses[0].split(".")
         datum["kVar"] = (
             float(datum["kW"])
             / float(datum["PF"])
             * math.sqrt(1 - float(datum["PF"]) * float(datum["PF"]))
         )
         datum["bus1"] = bus[0]
-        datum["numPhases"] = len(bus[1:])
+        datum["numPhases"] = dss.CktElement.NumPhases()
```

Comment:
Just checking, this doesn't include the neutral phase as well right? So nodename.1 and nodename.0.1 will both have NumPhases as 1?

Reply:
If a user has specified node 0, then I believe it might include it.

Reply:
ok, so I have tested this, and it will definitely include it, but it will not include bus.0 in the node name. I'm

Reply:
I think so yes. Otherwise a node with name.0.1.2.3 will end up with 4 phases instead of 3.

Reply:
Ok, I've added a test for this and a fix.